cmdstanpy/utils.py

```diff
@@ -933,6 +935,20 @@ def __init__(self, *objs: Union[str, dict], logger: logging.Logger = None):
                 if not os.path.exists(obj):
                     raise ValueError("File doesn't exist {}".format(obj))
                 self._paths[i] = obj
+            elif isinstance(obj, list):
+                if not all(isinstance(obj_item, str) for obj_item in obj):
+                    raise ValueError("Not all list items are str")
+                missing_obj_items = []
+                for obj_item in obj:
+                    if not os.path.exists(obj_item):
+                        missing_obj_items.append(obj_item)
+                if missing_obj_items:
+                    raise ValueError(
+                        "File doesn't exist: {}".format(
```

Comment:
Better error msg?

Reply:
if the calling code provides additional context, then this should be OK - there are a couple of kinds of args that can be either a dict or a json file, or list thereof.